src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -791,8 +791,21 @@ x3dom.registerNodeType(
                         hasColor = false;
                     }
                     this._mesh._numColComponents = numColComponents;
+                    if (fieldName.startsWith("attrib")) {
+                        var attrName = fieldName.slice(7);
+                        var attrNode = this._cf.attrib.nodes.find(
+                            function (a) {return a._vf.name == attrName;}
+                        );
```

Comment:
There are no way to know which attribute changed in the FieldChanged of IndexedFaceSet, and this is made specialy to avoid to update/compute all the other attributes.